k8s/openebs-node-exporter.json

```diff
@@ -1615,7 +1702,10 @@
       {
         "allFormat": "glob",
         "allValue": null,
-        "current": {},
+        "current": {
```

Comment:
should this be empty?

Reply:
yes @kmova , (came from improper cleanup after downloading modified json from grafana). Have made it empty.